packages/esm-service-queues-app/src/side-menu/side-menu.test.tsx

```diff
@@ -5,7 +5,6 @@ import SideMenu from './side-menu.component';
 jest.mock('@openmrs/esm-framework', () => ({
```

Comment:
I was going to point out that `LeftNavMenu` is [already mocked](https://github.com/openmrs/openmrs-esm-core/blob/4d3f1b0e33f51f805ef75ac81fe9d8e849e6e931/packages/framework/esm-framework/mock.tsx#L61), but actually I think this is maybe a useless test and we would be better off just deleting this file.